docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md

```diff
@@ -1,125 +1,98 @@
+You need to configure a custom values file for the Helm Chart of ScalarDL Ledger and ScalarDL Schema Loader (for Ledger) based on your environment (e.g., access information of the database you created in **Step 4**). See [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md) for more details.
 
-* [Configure a custom values file of Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+## Step 10. Deploy ScalarDL Ledger using Scalar Helm Chart
 
-## Step 12. Deploy Scalar DL Auditor using Scalar Helm Chart
+Deploy ScalarDL Ledger on your AKS cluster using Scalar Helm Chart. See [Deploy Scalar products using Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/how-to-deploy-scalar-products.md) for more details.
 
-Deploy Scalar DL Auditor on your AKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
+Note: We recommend creating a dedicated namespace by using the `kubectl create ns scalardl-ledger` command and deploying ScalarDL Ledger in the namespace by using the `-n scalardl-ledger` option with the `helm install` command.
 
-* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+## Step 11. Prepare a custom values file of Helm Chart of ScalarDL Auditor
 
-## Step 13. Check the status of Scalar DL Ledger deployment
+You need to configure a custom values file for the Helm Chart of ScalarDL Auditor and ScalarDL Schema Loader (for Auditor) based on your environment (e.g., access information of the database you created in **Step 5**). See [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md) for more details.
 
-After deploying Scalar DL Ledger on your AKS cluster, you need to check the status of each component. Please refer to the following document for more details.  
+## Step 12. Deploy ScalarDL Auditor using Scalar Helm Chart
 
-* [What you might want to check on a regular basis](./RegularCheck.md) // TODO: Update existing document
+Deploy ScalarDL Auditor on your AKS cluster using Scalar Helm Chart. See [Deploy Scalar products using Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/how-to-deploy-scalar-products.md) for more details.
 
-## Step 14. Check the status of Scalar DL Auditor deployment
+Note: We recommend creating a dedicated namespace by using the `kubectl create ns scalardl-auditor` command and deploying ScalarDL Auditor in the namespace by using the `-n scalardl-auditor` option with the `helm install` command.
```

Comment:
I added a new recommendation based on the offline discussion. We recommend using the dedicated k8s namespace for Scalar product deployment.